crates/rome_js_analyze/src/analyzers/nursery/no_header_scope.rs

```diff
@@ -0,0 +1,101 @@
+use rome_analyze::context::RuleContext;
+use rome_analyze::{declare_rule, ActionCategory, Ast, Rule, RuleDiagnostic};
+use rome_console::markup;
+use rome_diagnostics::Applicability;
+use rome_js_syntax::{JsxAttribute, JsxOpeningElement, JsxSelfClosingElement};
+use rome_rowan::{declare_node_union, AstNode, BatchMutationExt};
+
+use crate::JsRuleAction;
+
+declare_rule! {
+    /// Enforce scope prop is only used on `th` elements.
```

Comment:
```suggestion
    /// Check that the scope attribute is only used on `th` elements.
```

The old documentation was poorly worded... 😓